README.md

```diff
@@ -1,5 +1,14 @@
 # ChromeDragon
 
+Group: KI-47
+Student: Boiko Yurii
+Variant: 3
+HW i-face: I2C
+Game: chrome dragon game
+Data driven format: BINARY
+
+Project: Messenger with translating(Bachelor work)
```

Comment:
If you are planing to do your project then I need to have confirmtion from your mentor

Reply:
Confirmed with mentor (Oberyshyn)